### ISSUE TITLE
issues and enhancement for parameter plev for zonal mean 2d diags

### ISSUE THREAD

**Issue by chengzhuzhang** [E3SM-Project/e3sm_diags#265]
Customize plev works when specifying plev as follows:
```
z2d_param = ZonalMean2dParameter()
new_plevs = [1000.0, 900.0, 800.0, 700.0, 600.0, 500.0, 400.0, 300.0, 200.0, 100.0, 70.0, 50.0, 30.0, 20.0, 10.0]
z2d_param.plevs = new_plevs[::-1]
```
It doesn't work setting plevs, using two following methods:
```
z2d_param.plevs = list(numpy.logspace(1.0, 3.0, num=17))
```
or
```
z2d_param.plevs = [10.0, 13.33521432163324, 17.78279410038923, 23.71373705661655, 31.622776601683793, 42.169650342858226, 56.23413251903491, 74.98942093324558, 100.0, 133.3521432163324, 177.82794100389228, 237.13737056616552, 316.22776601683796, 421.6965034285823, 562.341325190349, 749.8942093324558, 1000.0]
```
or
```
z2d_param.plevs =numpy.logspace(1.0, 3.0, num=17)
```
getting errors:
```Traceback (most recent call last):
  File "run_diags_plevs.py", line 26, in <module>
    runner.run_diags([param,z2d_param])
  File "/lcrc/soft/climate/e3sm-unified/base/envs/e3sm_unified_1.3.0/lib/python3.7/site-packages/acme_diags/run.py", line 26, in run_diags
    final_params = self.get_final_parameters(parameters)
  File "/lcrc/soft/climate/e3sm-unified/base/envs/e3sm_unified_1.3.0/lib/python3.7/site-packages/acme_diags/run.py", line 51, in get_final_parameters
    self._add_parent_attrs_to_children(parameters)
  File "/lcrc/soft/climate/e3sm-unified/base/envs/e3sm_unified_1.3.0/lib/python3.7/site-packages/acme_diags/run.py", line 128, in _add_parent_attrs_to_children
    nondefault_param_child = self._find_attrs_with_nondefault_values(parameters[i])
  File "/lcrc/soft/climate/e3sm-unified/base/envs/e3sm_unified_1.3.0/lib/python3.7/site-packages/acme_diags/run.py", line 202, in _find_attrs_with_nondefault_values
    getattr(new_instance, attr) != getattr(param, attr):
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

@oksanaguba requested an enhancement to display pressure level on log-scale

**Comment by chengzhuzhang** [comment]
Also to add to check-value that plevs parameter needs to be increasing monotonically

**Comment by chengzhuzhang** [comment]
The error aforementioned was caused by the fact that the input parameter plevs is a numpy array. When comparing two numpy arrays, it returns a boolean array. To enforce plevs to be a list resolve this problem.

**Comment by chengzhuzhang** [comment]
Resolved in PR#266 after `e3sm_diags v2`. Following is a example python script to configure such a run, with plevs specified:
```
import os
import numpy
from acme_diags.parameter.core_parameter import CoreParameter
from acme_diags.parameter.zonal_mean_2d_parameter import ZonalMean2dParameter
from acme_diags.run import runner

param = CoreParameter()
param.reference_data_path = '/lcrc/group/acme/onguba/acme_scratch/anvil/preqx-ne30-default-nov01/run/climoout.86790-0002-0006'
param.test_data_path = '/home/onguba/as/preqx-tom-dec23/run/climoout.109811-0002-0006'
prefix = '/lcrc/group/acme/public_html/diagnostic_output/zhang40'
param.results_dir = os.path.join(prefix, 'zonal-vs-default-custom-plev-log')
param.test_title = 'preqx-tom-dec23.109811'
param.run_type = 'model_vs_model'
param.multiprocessing = True
param.num_workers =  8
z2d_param = ZonalMean2dParameter()
z2d_param.plevs =  [1000.0, 900.0, 800.0, 700.0, 600.0, 500.0, 400.0, 300.0, 200.0, 100.0, 70.0, 50.0, 30.0, 20.0, 10.0]
z2d_param.plot_log_plevs = True


runner.sets_to_run = ['zonal_mean_2d']
runner.run_diags([param,z2d_param])
```
